### PR TITLE
Fix some pylint issues

### PR DESCRIPTION
**Pull request by michalfabik**
This fixes whitespace issues I ran into while reviewing #356 and some other easy ones.  
  
See commits.

### CODE REVIEW

File: retrace-server.spec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Summary: Application for remote coredump analysis
 Name: retrace-server
-Version: 1.21.0
+Version: 1.21.0.159+gc34c
```
Comment (xsuchy) [inline retrace-server.spec:3]: Why this?

**Comment by xsuchy** [comment]
I am afraid that the last commit ba31850c0ab82f07476e5155fcadbc5f827bf858 contains something totally different that the description states. :(

File: retrace-server.spec (hunk 1 above)
Comment (michalfabik) [reply retrace-server.spec]: Sorry, accident.

File: src/retrace/hooks/hooks.py [hunk 2]
```diff
@@ -99,7 +99,7 @@ def _process_script(self, hook: str, hook_path: Path, exc_path: str) -> None:
 
         try:
             child = run(cmd, shell=True, timeout=hook_timeout, cwd=hook_path,
-                        stdout=PIPE, stderr=PIPE, encoding='utf-8')
+                        stdout=PIPE, stderr=PIPE, encoding='utf-8', check=False)
             child.check_returncode()
```
Comment (mgrabovsky) [inline src/retrace/hooks/hooks.py:103]: We could just shorten this to
```suggestion
                        stdout=PIPE, stderr=PIPE, encoding='utf-8', check=True)
```
since we're calling `check_returncode()` afterwards anyway.

Comment (michalfabik) [reply src/retrace/hooks/hooks.py]: Thanks, fixed.
